aws/resource_aws_rds_cluster_test.go

```diff
@@ -1651,6 +1677,54 @@ resource "aws_db_subnet_group" "test" {
 `, bucketName, bucketPrefix, uniqueId, uniqueId, uniqueId, bucketPrefix, uniqueId, uniqueId, uniqueId, uniqueId)
 }
 
+func testAccAWSClusterConfig_pointInTimeRestoreSource(parentId, childId string) string {
+	return fmt.Sprintf(`
+data "aws_availability_zones" "available" {}
+
+resource "aws_rds_cluster" "test" {
+  cluster_identifier = "%s"
+  master_username = "root"
+  master_password = "password"
+  db_subnet_group_name = "${aws_db_subnet_group.test.name}"
+  skip_final_snapshot = true
+  engine = "aurora-mysql"
+}
+
+resource "aws_vpc" "test" {
```

Comment:
not necessary per comment above

Reply:
`db_subnet_group_name` is a valid param according to the [API docs](https://docs.aws.amazon.com/AmazonRDS/latest/APIReference/API_RestoreDBClusterToPointInTime.html), and also documented as existing functionality in [terraform docs](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/rds_cluster#db_subnet_group_name). 

I've also validated that it works fine in test. Do you think you might have hit another problem that caused terraform to show an error related to the subnet group param?

Reply:
hmm not sure what I was running into that first go around but test looks good atm 👍